src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -426,11 +425,17 @@ void AdamsBashforthN::update_u_impl(const gsl::not_null<UpdateVars*> u,
                                     const History<Vars, DerivVars>& history,
                                     const Delta& time_step,
                                     const size_t order) const noexcept {
-  ASSERT(history.size() <= order_,
+  ASSERT(history.size() <= order_ + 1,
          "Length of history (" << history.size() << ") "
-         << "should not exceed target order (" << order_ << ")");
+         << "should not exceed target order (" << order_ << ") by more than 1");
```

Comment:
What is this change about?

Reply:
Ah, yeah good catch -- these couple of lines of the hunk should be dropped. It was for an earlier and less elegant attempt at the history cleanup change.